packages/compiler/core/checker.ts

```diff
@@ -2132,6 +2190,13 @@ export function createChecker(program: Program): Checker {
     // Evaluate the properties after
     checkModelProperties(node, type.properties, type, mapper);
 
+    for (const prop of walkPropertiesInherited(type)) {
+      const table = augmentedSymbolTables.get(node.symbol.members!) ?? node.symbol.members!;
```

Comment:
If we're going to mutate, then we shouldn't use a symbol table directly on a node. I think this should either be getOrCreateTable (forget exact name) or assert that we already have a table if we expect that something else created it.

Reply:
yeah, let me check on that.